benchmarks/runners/paper_lha_bench.py

```diff
@@ -125,24 +136,23 @@ class BenchmarkFFNS(LHABenchmark):
             "ktThr": np.inf,
         }
     )
-    is_FFNS = True
 
-    def ffns_nnlo_pdfs(self):
-        """
-        Set different pdfs set for NNLO FFNS
-        """
-        self.skip_pdfs.extend([-5, 5, "T24"])
-        if "V8" in self.skip_pdfs:
-            self.skip_pdfs.remove("V8")
+    def get_skip_pdfs(self, theory_updates):
```

Comment:
In general naming things `get_*` it's not really pythonic, since usually they are used in other languages (like Java) for accessing attributes, while Python uses `@property` to virtualize an attribute.

In this case you don't have a virtual attribute but really a function, nevertheless you could wish to avoid the `get_*` pattern, and the easiest way it's simply to drop `get_` and retain `*` (`skip_pdfs` in this case) since `get_` it's not adding anything.
It's really a minor aesthetic issue in this case.

Reply:
yes, I know - Problem is, we can not use just `skip_pdf`, I believe, since the attribute is already called this way and I need to pass directly some parameters (i.e. theory) to it